lib/error.js

```diff
@@ -3,12 +3,6 @@
 const MongoNetworkError = require('./core').MongoNetworkError;
 const mongoErrorContextSymbol = require('./core').mongoErrorContextSymbol;
 
-const GET_MORE_NON_RESUMABLE_CODES = new Set([
```

Comment:
if we are removing this, is it useful to have the `isGetMoreError` message below? If that's not required, can we also remove the `mongoErrorContextSymbol` outright?

Reply:
You're right, we don't need it anymore.